HW6/scripts/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu
+
+RUN mkdir /scripts
+RUN mkdir /results
+
+COPY ./Homework_6.sh /scripts/Homework_6.sh
+COPY ./baseball.sql /scripts/baseball.sql
+COPY ./Homework_6.sql /scripts/Homework_6.sql
+RUN chown 1000:1000 /scripts
```

Comment:
Minor question: NOT a COMMENT

Why is the 1000:1000 used here ?  Isn't this already a copy operation from host to container. So container will, by default have the user ownerships of the directories.